rocAL/include/api/rocal_api_types.h

```diff
@@ -483,4 +483,19 @@ struct RocalShardingInfo {
           shard_size(shard_size) {}
 };
 
+/*! \brief Missing components behaviour for Webdataset
+ *  \ingroup group_rocal_types
+ */
+enum RocalMissingComponentsBehaviour {
+    /*! \brief ROCAL_ERROR
+     */
+    ROCAL_THROW_ERROR = 0,
```

Comment:
I suggest renaming to more specific "ROCAL_MISSING_COMPONENT_ERROR" etc..

Reply:
Renaming done